entsoe-util/src/main/java/com/powsybl/entsoe/util/MergedXnode.java

```diff
@@ -19,13 +19,33 @@ default String getName() {
         return "mergedXnode";
     }
 
-    float getRdp();
+    default float getRdp() {
+        return 0.5f;
+    }
+
+    default MergedXnode setRdp(float rdp) {
+        return this;
+    }
+
+    default float getXdp() {
+        return 0.5f;
+    }
+
+    default MergedXnode setXdp(float xdp) {
+        return this;
+    }
```

Comment:
You do not need to add a default implementation here (for this four methodes) in my opinion since it was already present in the API. We had a mistake in the in-memory implementation but it is not necessary the case in others. In any case, this default implementation will be overrided in other custom implementations

Reply:
Removed